db/migrate/20180501093846_create_events.rb

```diff
@@ -0,0 +1,39 @@
+# This migration does not use Rails ActiveRecord ORM DSL since
+# it doesn't provide data integrity (foreign key) for polymorphic models.
```

Comment:
I'm not sure I fully understand what you mean.

Reply:
Read the article :joy: